src/lib/util/octets.ts

```diff
@@ -0,0 +1,123 @@
+import { Field, checkBitLength } from '../provable/field.js';
```

Comment:
cherry picked from https://github.com/o1-labs/o1js/pull/1949

Reply:
this is weirdly placed, it's the only circuit code not located in `src/provable` or `src/mina`

also maybe it'd be more accessible as `field.toBytes()` and `Field.fromBytes()`?

Reply:
Actually the reason that it went to its own util file was that I was getting circular dependencies preventing me from calling it from `Field` directly, which was my first attempt. Seems like that has been solved here.